fcrepo-http-commons/src/test/java/org/fcrepo/http/commons/domain/ldp/LdpPreferTagTest.java

```diff
@@ -104,15 +103,6 @@ public void testPreferOmitContainmentAndMembership() throws ParseException {
         assertFalse(testObj.prefersContainment());
     }
 
-    @Test
-    public void testPreferEmbed() throws ParseException {
-        final PreferTag prefer
-                = new PreferTag("return=representation; include=\"" + EMBED_CONTAINS + "\"");
```

Comment:
Instead, can we just change the header to `EMBED_CONTAINED`?

Reply:
We've got a similar test testEmbedContained() in https://github.com/fcrepo4/fcrepo4/pull/1290/files/ef6e3ccf120bdc6c1ff36679db7cdccd689633b9#diff-1f9ac54b0c450cf5f9a4f7b2b6b94c3dL138 as well.